lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -339,12 +341,12 @@ def init_api_client
         key = Google::APIClient::PKCS12.load_key(@private_key_path,
                                                  @private_key_passphrase)
         jwt_asserter = Google::APIClient::JWTAsserter.new(
-          @private_key_email, 'https://www.googleapis.com/auth/logging.write',
-          key)
+          @private_key_email, LOGGING_SCOPE, key)
         @client.authorization = jwt_asserter.to_authorization
         @client.authorization.expiry = 3600  # 3600s is the max allowed value
       else
-        @client.authorization = Google::APIClient::ComputeServiceAccount.new
+        @client.authorization = Google::Auth.get_application_default(
```

Comment:
test_one_log_with_invalid_json_credentials causes this to throw an exception (Runtime Exception "Unable to read the credential file ...").

I might be able to add one more test for the case where we're not on GCE and didn't specify any credentials, but I need the other PR to enable me to do that without major changes... I can add it after I commit both of these.